src/compas_fab/backends/ros/backend_features/move_it_forward_kinematics.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from compas.utilities import await_callback
+
+from compas_fab.backends.backend_feature_interfaces import ForwardKinematics
+from compas_fab.backends.ros.backend_features.helpers import validate_response
+from compas_fab.backends.ros.messages import GetPositionFKRequest
+from compas_fab.backends.ros.messages import GetPositionFKResponse
+from compas_fab.backends.ros.messages import Header
+from compas_fab.backends.ros.messages import JointState
+from compas_fab.backends.ros.messages import MultiDOFJointState
+from compas_fab.backends.ros.messages import RobotState
+from compas_fab.backends.ros.planner_backend import ServiceDescription
+
+
+class MoveItForwardKinematics(ForwardKinematics):
+    GET_POSITION_FK = ServiceDescription('/compute_fk',
+                                         'GetPositionFK',
+                                         GetPositionFKRequest,
+                                         GetPositionFKResponse,
+                                         validate_response)
+
+    def __init__(self, ros_client):
+        self.ros_client = ros_client
+
+    def forward_kinematics(self, configuration, group=None, options={}):  # !!! must find all calls to this and adapt  GHX!!!
+        base_link = options['base_link']
+        link_name = options.get('link_name')
+        return self.forward_kinematics_deprecated(base_link, configuration, group, link_name)
+
+    def forward_kinematics_deprecated(self, base_link, configuration, group, ee_link):
```

Comment:
My intention for leaving these `*_deprecated` methods was for ease of transition for any user's existing scripts, that they would only have to change the method name rather than dig through and figure out exactly how to adapt all the signatures.  But then I went and changed the signatures of some of the deprecated methods, eg `robot` is replaced by `base_link`.  So I see a few possibilities here:  1. Leave it as it is, somewhat misleadingly. 2. Squash all the `*_deprecated` methods with their callers, and make the users work to adapt and leave little trace in the code of how things 'used to be'. 3.  Bootstrap a minimal `robot` instance from the actually required data like `base_link` within the new version of the functions and restore the `*_deprecated` method to its original signature.

Reply:
The 2nd and 3rd options sound best. I would slightly prefer the 3rd, but I fear it might take too much time for too little benefit.

Reply:
For the third option, is there a reason I shouldn't use something like 
```
robot = self.client.load_robot()
```
in, say, `plan_motion` to use as an argument in `plan_motion_deprecated`?

Reply:
As discussed earlier today, the 3rd option is going to be too much work for very little gain, let's go with 2 and maintain only a clean, new API.